src/components/MessageForm.tsx

```diff
@@ -22,8 +22,8 @@ const initValues: MessageFormData = {
 
 const validationSchema = Yup.object<YupShape<MessageFormData>>({
   message: Yup.string().required(
-    i18n.t('feedback.validation.is_required', {
-      field: i18n.t('common.message'),
+    i18n.t('translation:feedback.validation.is_required', {
+      field: i18n.t('translation:common.message'),
```

Comment:
Det er noe kluss som gjør at man må angi namespace når man bruker i18n-objektet direkte (ikke via hook). "translation" er default namespace, så burde ikke være nødvendig, men finner ikke ut hvorfor det blir slik. Ikke noe stort problem så ser mellom fingrene på dette for nå.